airgun/entities/subnet.py

```diff
@@ -36,3 +40,15 @@ def step(self, *args, **kwargs):
         self.view.browser.wait_for_element(
             self.parent.new, ensure_page_safe=True)
         self.parent.browser.click(self.parent.new)
+
+
+@navigator.register(SubnetEntity, 'Edit')
+class EditExistingSubnet(NavigateStep):
```

Comment:
Why `EditExistingSubnet` and not just `EditSubnet`?
I can see some other entities are following the same naming pattern, so probably not a blocker since we'll need some extra effort to fix it elsewhere.